src/Compilers/Core/Portable/MetadataReader/PEModule.cs

```diff
@@ -1199,6 +1199,27 @@ internal bool HasMaybeNullWhenOrNotNullWhenOrDoesNotReturnIfAttribute(EntityHand
             return false;
         }
 
+        internal ImmutableHashSet<string> GetStringValuesOfNotNullIfNotNullAttribute(EntityHandle token)
+        {
+            var attributeInfos = FindTargetAttributes(token, AttributeDescription.NotNullIfNotNullAttribute);
```

Comment:
You can use the `FindTargetAttribute` version from this type.

Reply:
@333fred Don't we want all the attributes?

Reply:
Oh, interesting, I didn't realize that `NotNullIfNotNull` had `AllowMultiple = true`. In that case, never mind.